src/Compilers/CSharp/Test/Syntax/Parsing/ParserRegressionTests.cs

```diff
@@ -199,8 +199,8 @@ public void LexerTooManyBadTokens()
             var parsedTree = ParseWithRoundTripCheck(source.ToString());
             IEnumerable<Diagnostic> actualErrors = parsedTree.GetDiagnostics();
             Assert.Equal("202", actualErrors.Count().ToString());
-            Assert.Equal("(1,1201): error CS1056: Unexpected character '\\u003C'", actualErrors.ElementAt(200).ToString());
```

Comment:
>ToString [](start = 111, length = 8)

Rather than calling Object.ToString(), we should call IFormattable.ToString and passing EnsureEnglishUICulture.PreferredOrNull as the culture